versioned_docs/version-v0.53/api/public-endpoints.md

```diff
@@ -9,9 +9,11 @@ import DataNodes from '@site/src/components/DataNodes';
 
 Due to the distributed nature of Vega, most of the APIs are served from a data node, and a few from a core node. This means there is no single API server, and users will need to choose a node to connect to.
 
-This differs from centralised services, as you can't connect to an API server run by a single company, nor does it require a specific access token. 
+This differs from centralised services, as you can't connect to an API server run by a single company, nor does it require a specific access token.
 
-Below, find a list of all the public endpoints available for this network. 
+Validators, community members, and other service providers may operate public data nodes. These may have usage restrictions, rate limits, limited data retention, or other restrictions or terms. Check with an individual operator if you have specific needs or questions.
+
+Below, find a list of the known public endpoints available for this network. 
```

Comment:
Pretty sure this is duplicatred and you say pretty much the exact same thing a bit before.

Reply:
Different page I believe, and I don't expect every reader to read every page (unfortunately)